src/applications/vaos/components/calendar/CalendarCell.jsx

```diff
@@ -106,14 +105,13 @@ const CalendarCell = ({
         type="button"
         ref={buttonRef}
       >
-        {inSelectedArray && (
-          <CalendarSelectedIndicator
-            date={date}
-            fieldName={id}
-            selectedDates={selectedDates}
-            selectedIndicatorType={selectedIndicatorType}
-          />
-        )}
+        {inSelectedArray &&
```

Comment:
This is the core change, switching between the checkmark indicator and the results of the renderIndicator function